packages/wdio-types/src/Capabilities.ts

```diff
@@ -5,83 +5,119 @@ import type {
 
 type JSONLike = | { [property: string]: JSONLike } | readonly JSONLike[] | string | number | boolean | null
 
-export type PageLoadingStrategy = 'none' | 'eager' | 'normal';
+export type PageLoadingStrategy = 'none' | 'eager' | 'normal'
 export type LoggingPreferenceType =
     'OFF' | 'SEVERE' | 'WARNING' |
     'INFO' | 'CONFIG' | 'FINE' |
-    'FINER' | 'FINEST' | 'ALL';
+    'FINER' | 'FINEST' | 'ALL'
 
 export interface LoggingPreferences {
-    browser?: LoggingPreferenceType;
-    driver?: LoggingPreferenceType;
-    server?: LoggingPreferenceType;
-    client?: LoggingPreferenceType;
+    browser?: LoggingPreferenceType
+    driver?: LoggingPreferenceType
+    server?: LoggingPreferenceType
+    client?: LoggingPreferenceType
 }
 
-export type Timeouts = Record<'script' | 'pageLoad' | 'implicit', number>;
+export type Timeouts = Record<'script' | 'pageLoad' | 'implicit', number>
 
-export type ProxyTypes = 'pac' | 'noproxy' | 'autodetect' | 'system' | 'manual';
+export type ProxyTypes = 'pac' | 'noproxy' | 'autodetect' | 'system' | 'manual'
 
 export interface ProxyObject {
-    proxyType?: ProxyTypes;
-    proxyAutoconfigUrl?: string;
-    ftpProxy?: string;
-    ftpProxyPort?: number;
-    httpProxy?: string;
-    httpProxyPort?: number;
-    sslProxy?: string;
-    sslProxyPort?: number;
-    socksProxy?: string;
-    socksProxyPort?: number;
-    socksVersion?: string;
-    socksUsername?: string;
-    socksPassword?: string;
-    noProxy?: string[];
+    proxyType?: ProxyTypes
+    proxyAutoconfigUrl?: string
+    ftpProxy?: string
+    ftpProxyPort?: number
+    httpProxy?: string
+    httpProxyPort?: number
+    sslProxy?: string
+    sslProxyPort?: number
+    socksProxy?: string
+    socksProxyPort?: number
+    socksVersion?: string
+    socksUsername?: string
+    socksPassword?: string
+    noProxy?: string[]
+}
+
+export interface CapabilityOptions {
```

Comment:
Can we remove this interface and merge everything into [`WebdriverIOCapabilities`](https://github.com/webdriverio/webdriverio/blob/0652d3c0212256108d3f505f98b95d97a5b5f591/packages/wdio-types/src/Capabilities.ts#L213)?

Reply:
kk, done